service/lib/agama/autoyast/scripts_reader.rb

```diff
@@ -85,6 +86,8 @@ def read_post_scripts
         scripts = scripts_section.fetch("chroot-scripts", []).map do |script|
           read_post_script(script)
         end
+        scripts += ServicesManagerReader.new(profile).read
```

Comment:
If you want to change the array in place, you might want to use `concat`. But I do not have a strong opinion.